environment.yml

```diff
@@ -13,9 +13,8 @@ dependencies:
   - pandas
 
 # C++ toolchain
-  - clang=13.0.1
-  - clangxx=13.0.1
-  - clang-tools=13.0.1
+  - cxx-compiler 
```

Comment:
switched out clang/clangxx for cxx-compiler since `cxx-compiler` gets you the appropriate compiler for each platform such that everything is ABI compatible with other conda-forge packages. that is, it gives the right version on each of:
* gcc on linux
* clang on mac
* vc on windows